cookiecutter-django-ida/{{cookiecutter.repo_name}}/Makefile

```diff
@@ -181,18 +181,14 @@ db-shell: # Run the app shell as root, enter the app's database
 %-attach:
 	docker attach {{cookiecutter.project_name}}.$*
 
-github_docker_build:
-	docker build . -f Dockerfile --target app -t openedx/{{cookiecutter.repo_name}}
-
-github_docker_tag: github_docker_build
-	docker tag openedx/{{cookiecutter.repo_name}} openedx/{{cookiecutter.repo_name}}:${GITHUB_SHA}
-
-github_docker_auth:
+docker_auth:
 	echo "$$DOCKERHUB_PASSWORD" | docker login -u "$$DOCKERHUB_USERNAME" --password-stdin
 
-github_docker_push: github_docker_tag github_docker_auth ## push to docker hub
-	docker push 'openedx/{{cookiecutter.repo_name}}:latest'
-	docker push "openedx/{{cookiecutter.repo_name}}:${GITHUB_SHA}"
+docker_tag_build_push: docker_auth
+	docker buildx build -t openedx/{{cookiecutter.repo_name}}:latest -t openedx/{{cookiecutter.repo_name}}:${GITHUB_SHA} --platform linux/amd64,linux/arm64 --target production --push .
+
+docker_tag_build_push_dev: docker_auth
+	docker buildx build -t openedx/{{cookiecutter.repo_name}}-dev:latest -t openedx/{{cookiecutter.repo_name}}-dev:${GITHUB_SHA} --platform linux/amd64,linux/arm64 --target development --push .
```

Comment:
You have defined the target in Dockerfile as `dev` so this should be `dev` as well.

Reply:
Ooh, that's a good catch! Same for prod/production.

Reply:
Thanks @iamsobanjaved. It's fixed.